docs/FAQ.md

```diff
@@ -18,10 +129,9 @@ intermediary loaders with a common Linux-based loader that provides a rich
 security processing framework. One role that TrenchBoot does not fulfill is
 that the UEFI shim also serves as a trust delegation point that transitions
 from Microsoft Authority to Distribution/Installer/No Authority. The response
-why this is not of concern will be addressed in Question 2.
+why this is not of concern will be addressed in the next question.
 
-
-## 2. What are the benefits of measurement over signature validation?
+## 4. What are the benefits of measurement over signature validation?
```

Comment:
I would not say "What are the benefits of measurement over signature validation?", I would say "How do measurement trust chains contrast with verification trust chains?".